src/lib/schemeUtils.ts

```diff
@@ -148,6 +148,24 @@ export function schemeNameAndAddress(address: string) {
   }
 }
 
+/**
+ * given the address (of a scheme), return scheme's name
+ * @param  address [description]
+ * @return         [description]
+ */
+export function schemeNameFromAddress(address: string) {
```

Comment:
Can you please refactor `schemeNameAndAddress` (just above) to use this new function?